server/pom.xml

```diff
@@ -244,7 +244,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven.compiler.plugin.version}</version>
                 <configuration>
-                    <release>${java.version}</release>
```

Comment:
I see that `java.version` is used only here, so you can change it to "1.8" and reuse for `source` and `target`